### PR TITLE
gsettings-desktop-schemas: Nuke overrides

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**
- All of these overrides should be in branding packages unless the default is super stupid.

**Test Plan**

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by joebonrichie** [comment]
Need ack from @EbonJaeger on budgie branding before merging, i'll handle gnome branding. Mate is deprecated.

**Comment by EbonJaeger** [comment]
I added the used overrides to the Budgie desktop branding source package. I would have added an update to the package in this branch, but the branch is out-of-date with the latest changes.

**Review by HarveyDevel — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
